content/post/glue-work.md

```diff
@@ -0,0 +1,79 @@
+- Documentation
+- Interviewing
+- Training business users on tools or data sources
+- Building the [culture of partnership](https://www.locallyoptimistic.com/post/culture_of_partnership/)
+- [Communicating well](https://www.locallyoptimistic.com/post/bad_communication/) within the analytics team and across the company
+- Unblocking a teammate
+- Responding to ad-hoc questions and triaging inbound requests
+- Building subject matter expertise
+- Creating a pull request template for the team
+- Trialing a new collaboration tool and making a recommendation to the team
+- Anything that affects how your team works instead of the work itself
+Sometimes within analytics teams, it’s work that crosses job descriptions. It can be:
+- The analyst who implements automated testing
+- The engineer who reaches out directly to a sales rep to understand how the data they are scraping ultimately gets used so they can improve the quality
+- The junior team member who remembers what a debacle their onboarding was, and creates a new onboarding plan for the next hire
+And sometimes, technical data folks report into a less technical manager, or have to pitch their promotion recommendations to a less technical leader. In that case, technical work can become glue work, including:
+- Designing processes and systems that scale well
+- Implementing monitoring and alerts for data freshness or quality
+- Improving query performance
+- Anything that is not immediately visible to end users of a reporting or visualization tool
+All of this work is important and valuable. And all of it belongs in an analytics team. However, there is a difference between *valuable* and *valued*. If there’s nothing in your job description or career ladder that addresses this work, when it comes to promo time, it can be hard to get credit for the time you spend on these tasks.  If glue work makes the team function more smoothly and effectively, but it is not valued, then that is a problem.
+
+## Why It Matters
+All the work listed above is truly important. It ensures that analysts' work is efficient, relevant, impactful, and scalable. Spending a week on a really interesting analysis that turns out not to be actionable - for example, recommending a change to operations that we can’t implement due to existing contract terms - is a waste of time and doesn’t make the analytics team look good. That analyst should be rewarded for spending time up front really understanding the situation and the boundaries. Spending extra time writing tests every time you write a transformation job (or spending a week writing lots of tests for existing jobs!) isn’t exciting to most people in the company, but is also really important to our ability to develop quickly and reliably. I lead analytics teams, and I *want* my team to be rewarded for glue work so we can all be more successful. 
```

Comment:
Hmmm. I agree with this for sure, but I get the sense from this paragraph that glue work is anything that isn't visible to non-analytics folks. For example, scoping projects is something I certainly would categorize as "core work that an analyst must do" that I wouldn't categorize as "tangential work that an analyst may or may not do", which is where you may tend to find more glue work. 

My concern is that glue work becomes too broadly applied. But I could be wrong!

Reply:
I have struggled with this throughout writing it. The definition of glue work varies so much across orgs because the standards of what good analytics looks like varies so much. I think this section can be clearer regardless.